### DISCUSSION TITLE
Identifying Modes

### DISCUSSION THREAD

**Discussion by C4E5G3** [helix-editor/helix#10897] [General]
Would it be possible to have a ball, a circle, to the left of the numbers. The ball would change colour with the mode. So if I am on line 4, in Normal Mode, there is a lime green ball on line 4 to the left of the 4, and my cursor bar is lime green. I switch to insert and the ball goes crimson, (and the cursor slat), select and it goes purple, etc.

**Comment by kirawi** [comment]
There's already a way to identify modes in Helix, so I don't think something like this would be implemented in core. Themes can highlight `statusline.select/insert/normal`
